kafka_utils/kafka_cluster_manager/cmds/command.py

```diff
@@ -200,10 +200,6 @@ def get_reduced_assignment(
             if set(replica) != set(new_assignment[t_p])
         ]
 
-        if (leaders_changes <= max_leader_only_changes and
-                partition_change_count <= max_partition_movements):
-            return new_assignment
```

Comment:
I think tests pass because that's just a small optimization. I'm ok with removing it.